docs/whats_new.rst

```diff
@@ -38,6 +38,7 @@ Enhancements
 - Add ``offset`` arg to :function:`braindecode.preprocessing.preprocess` (:gh:`599` by `Pierre Guetschel`_)
 - Add type hints to preprocessing (:gh:`600` by `Pierre Guetschel`_)
 - Add ``mypy`` type checks to pre-commit and CI (:gh:`606` by `Pierre Guetschel`_)
+- Add `"recording"` to `keys` in `RecordingSampler`; docs fix (:gh:`614` by `John Muradeli`_)
```

Comment:
Hi @OverLordGoldDragon,

Can you add your name to the bottom of this file?

Reply:
I work with the assumption that [612](https://github.com/braindecode/braindecode/pull/612/files) is merged, can adjust if not. But also https://github.com/braindecode/braindecode/pull/615#issuecomment-2117739226